### PR TITLE
update aws oidc db enrollment role permissions

### PR DESCRIPTION
**Pull request by GavinFrazar**
This PR adds the permissions necessary for the AWS RDS enrollment created IAM role to list VPCs and subnets.
Related PRs:
- https://github.com/gravitational/teleport/pull/44313
- https://github.com/gravitational/teleport/pull/44183

Part of:
- https://github.com/gravitational/teleport/issues/41004

cc @kimlisa

### CODE REVIEW

**Review by marcoandredinis — APPROVED** [review]

File: web/packages/teleport/src/Discover/Shared/Aws/ConfigureIamPerms.tsx [hunk 1]
```diff
@@ -142,7 +142,9 @@ export function ConfigureIamPerms({
       "Action": [
         "rds:DescribeDBInstances",
         "rds:DescribeDBClusters",
-        "ec2:DescribeSecurityGroups"
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeVpcs"
```
Comment (kimlisa) [inline web/packages/teleport/src/Discover/Shared/Aws/ConfigureIamPerms.tsx:147]: thank you! 👍

**Review by kimlisa — APPROVED** [review]

**Comment by marcoandredinis** [comment]
What happens if the user doesn't have the correct permissions?
Eg, let's say they ran the oneoff script that adds the required permissions before this was released.

Are they asked to re-run the script in cloudshell? Or do we just throw an error because they can list databases but can't list VPCs?

**Comment by GavinFrazar** [comment]
> What happens if the user doesn't have the correct permissions? Eg, let's say they ran the oneoff script that adds the required permissions before this was released.
> 
> Are they asked to re-run the script in cloudshell? Or do we just throw an error because they can list databases but can't list VPCs?

That's a good point and something we have to handle in the frontend code.
cc @kimlisa does https://github.com/gravitational/teleport/pull/44671 handle that already?

**Comment by kimlisa** [comment]
> > What happens if the user doesn't have the correct permissions? Eg, let's say they ran the oneoff script that adds the required permissions before this was released.
> > Are they asked to re-run the script in cloudshell? Or do we just throw an error because they can list databases but can't list VPCs?
> 
> That's a good point and something we have to handle in the frontend code. cc @kimlisa does #44671 handle that already?

i'm pretty sure it does, but i'll test it

**Pull request merged**